src/js/script.js

```diff
@@ -1,5 +1,3 @@
-import 'babel-polyfill';
-
```

Comment:
Fair call @feimosi, thank you! 
But even with preset env `babel-polyfill` adds almost 100kB to the bundle (and believe I tried lots of combinations). And since the only things we need polyfill now is Array.from I imported it from `core-js` directly. It's a bit more cumbersome, but thanks to that (and removing `underscore`) bundle size dropped to 19kB.